catalog/src/main/kotlin/com/adevinta/spark/catalog/model/Components.kt

```diff
@@ -91,8 +91,34 @@ public data class Component(
 private var nextId: Int = 1
```

Comment:
Just wondering, what are these id used for? if it is to reference unique things, we should use String keys. Also, I think these 2 lines could be removed now.

Reply:
I removed and used string instead of int for the ids in a6915cfe739196b1b581040379926f45e3b887aa